data/mods/Magiclysm/itemgroups/spellbooks.json

```diff
@@ -139,7 +139,10 @@
       [ "bio_sneeze_beam", 50 ],
       [ "spell_scroll_banishment_lesser", 30 ],
       [ "spell_scroll_nova_flare", 25 ],
-      [ "spell_scroll_freezing_touch", 40 ]
+      [ "spell_scroll_freezing_touch", 40 ],
+      [ "spell_scroll_focused_bolt", 1 ],
```

Comment:
rarity 1 items really should get a tier bump. recognizing this, the spell could definitely be more powerful than the tier 3, but we shouldn't get too out of hand until we can have some reasonable gating or scaling in the mod.

Reply:
ill increase them to 4 for test purposes